ros/api/v1/hosts.py

```diff
@@ -25,30 +25,46 @@
 ]
 
 
+def default_queries(ident, additional_filter=None):
+    account_query = db.session.query(RhAccount.id).filter(RhAccount.account == ident['account_number']).subquery()
+    if additional_filter is None:
+        return db.session.query(System.id).filter(System.account_id.in_(account_query))
+    return db.session.query(System.id).filter(System.account_id.in_(account_query)).filter(additional_filter)
+
```

Comment:
Thanks for the modularization :raised_hands: 
Looks like only the `account_number` is required from `ident`, shall we pass just the account_number
in such a case?
Also, I think we can move the function to utils.py or some other file that would be appropriate

Reply:
@saltgen You mean directly passing `ident['account_number']` instead of `ident`? Because if so, in any case we have to fetch the account number from ident, so I don't foresee much of a difference.
Do let me know if that sounds okay or not?

Reply:
Agreed, there is not much real-time gain in this, but in terms of readability, IMO this looks clearer.

```
def default_queries(account_number: str, additional_filter: bool = None):
    ...
    ...
```

P.S. Added the type hints just to convey what I really mean, not required to add them.

Reply:
Done.